src/api/assignments/index.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "../../plugins/axios";
+
+class Assignments {
+  static async get() {
+    try {
+      const response = await axios.get("/assignments");
```

Comment:
return await ...

Reply:
where is the return?

Reply:
que reemplaces la linea 6 por: return await axios.get("/assignments");